lib/jekyll-redirect-from/generator.rb

```diff
@@ -16,7 +16,7 @@ def generate(site)
 
       # Must duplicate pages to modify while in loop
       (site.docs_to_write + site.pages.dup).each do |doc|
-        next unless JekyllRedirectFrom::CLASSES.include?(doc.class)
+        next unless JekyllRedirectFrom::CLASSES.any? { |klass| doc.is_a? klass }
```

Comment:
Let's put this into its own method.

```ruby
def redirectable_document?(doc)
  doc.is_a?(Jekyll::Document) || doc.is_a?(Jekyll::Page)
end
```

Then do `next unless redirectable_document?(doc)`